### PR TITLE
118/extract actions logic

### PR DESCRIPTION
**Pull request by anxolin**
Extracts the action logic out of the widget and move it to a hook.
This simplifies the widget that was getting a bit big.

This PR is in response to @Velenir comment:
https://github.com/gnosis/dex-react/pull/120#discussion_r336944597

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr142--dexreact.review.gnosisdev.com

File: src/components/DepositWidget/useRowActions.tsx [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+      mounted.current = false
+    }
+  }, [])
+
+  function _getToken(tokenAddress: string): TokenBalanceDetails {
+    return balances.find(({ address }) => address === tokenAddress)
+  }
+
+  function _updateToken(tokenAddress: string, updateBalances: Mutation<TokenBalanceDetails>): void {
+    setBalances(balances =>
```
Comment (Velenir) [inline src/components/DepositWidget/useRowActions.tsx]: You can move `if mounted`check here, then there'll be no need to include it in any other places in this file.

Or, and this is crazy, we can agree on #132 and implement one of the proposals.

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr142--dexreact.review.gnosisdev.com

File: src/components/DepositWidget/useRowActions.tsx (hunk 1 above)
Comment (anxolin) [reply src/components/DepositWidget/useRowActions.tsx]: Addressed in 5de6de3070cb18b1c4baf5b823c0f2c50e8f06d3

Yes, let's discuss the refactor in that issue.

**Pull request merged**